### PR TITLE
Add TrustyAI test to check meanshift request

### PR DESCRIPTION
**Pull request by adolfo-ab**
Adds a test to verify the request of a TrustyAI drift metric (meanshift)

## Description
In this test we send a metric request and verify that the response makes sense

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by lugi0 — APPROVED** [review]

File: tests/trustyai/drift/utils.py [hunk 1]
```diff
@@ -242,3 +251,34 @@ def _check_pods_ready_with_env() -> bool:
...
+        trustyai_service (TrustyAIService): The TrustyAI service instance to interact with.
+        token (str): Authentication token for the service.
+        metric_name (str): Name of the metric to request.
+        json_data (Any): JSON payload for the metric request.
+    """
+
+    response = TrustyAIServiceRequestHandler(token=token, service=trustyai_service, client=client).send_drift_request(
+        metric_name=metric_name, json=json_data
+    )
+    LOGGER.info(msg=f"TrustyAI metric request response: {json.dumps(json.loads(response.text), indent=2)}")
```
Comment (rnetser) [inline tests/trustyai/drift/utils.py:273]: do we need this log as info or only as error when failing?

Comment (adolfo-ab) [reply tests/trustyai/drift/utils.py:273]: I think it's helpful as info, it's not a huge payload and it happens only one, so there won't be log spamming

Comment (rnetser) [reply tests/trustyai/drift/utils.py:273]: helpful even if the test passes?

File: tests/trustyai/drift/utils.py [hunk 2]
```diff
@@ -242,3 +251,34 @@ def _check_pods_ready_with_env() -> bool:
...
+
+    assert response.status_code == http.HTTPStatus.OK, f"Unexpected status code: {response.status_code}"
+    assert response_data["timestamp"] != "", "Timestamp is empty"
+    assert response_data["type"] == "metric", "Incorrect type"
+    assert response_data["value"] != "", "Value is empty"
+    assert isinstance(response_data["value"], float), "Value must be a float"
+    assert response_data["specificDefinition"] != "", "Specific definition is empty"
+    assert response_data["name"] == metric_name, f"Wrong name: {response_data['name']}, expected: {metric_name}"
+    assert response_data["id"] != "", "ID is empty"
+    assert response_data["thresholds"] != "", "Thresholds are empty"
```
Comment (rnetser) [inline tests/trustyai/drift/utils.py:284]: aggregate failures and assert once; this way you get all failures and not exit on the 1st one

File: tests/trustyai/drift/utils.py (hunk 1 above)
Comment (adolfo-ab) [reply tests/trustyai/drift/utils.py:273]: Yes, I think we want to see this even if the test passes

File: tests/trustyai/drift/utils.py (hunk 2 above)
Comment (adolfo-ab) [reply tests/trustyai/drift/utils.py:284]: done

**Review by rnetser — APPROVED** [review]

**Pull request merged**